ci/rpm_test_daos_test_node.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -ex
+sudo yum -y install daos{,-client}-${DAOS_PKG_VERSION}
+sudo yum -y history rollback last-1
+sudo yum -y install daos{,-{server,client}}-${DAOS_PKG_VERSION}
+sudo yum -y install daos{,-tests}-${DAOS_PKG_VERSION}
+
+me=$(whoami)
+for dir in server agent; do
+  sudo mkdir /var/run/daos_$dir
+  sudo chmod 0755 /var/run/daos_$dir
+  sudo chown $me:$me /var/run/daos_$dir
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]